src/qibo/models/dbi/double_bracket.py

```diff
@@ -103,12 +102,12 @@ def off_diag_h(self):
 
     @property
     def off_diagonal_norm(self):
-        """Norm of off-diagonal part of H matrix."""
+        r"""Hilbert Schmidt norm of off-diagonal part of H matrix: \sqrt{A^\dag A}"""
```

Comment:
Trace is missing in the doc string:

`r"""Hilbert Schmidt norm of off-diagonal part of H matrix: \sqrt{Tr[A^\dag A]}"""`